debian/README.etc

```diff
@@ -0,0 +1,17 @@
+Configuring the Kolibri Debian package:
+
+Kolibri is normally configured in its own environment, for instance in
+its main configuration file `~/.kolibri/options.ini` or through command
+line options such as `kolibri start --port=1234`.
+
+If you want Kolibri to be configured in its external environment, you can
+set a number of environment variables.
```

Comment:
I don't understand the terminology used here - "its own environment" versus "its external environment"

Reply:
Imagine environments as layers with a clear hierarchy and responsibility. The Debian package is external to Kolibri, it's maintained separately with a separate release flow.

Whatever the Debian installer and the user of the Debian package defines in `/etc/kolibri`, this is defined in an external environment to be respected by Kolibri.. but Kolibri has nothing to do with `/etc/kolibri` and knows nothing about these files. The environment could be defined by an alternative system service, by a UWSGI configuration, a Windows installer, a Mac installer, some UI, an app etc.

That's why it's important to keep it external.

Reply:
okay that makes sense. The part that I didn't really understand then was "it's own environment", because in some sense all environments are external...

But in any case, the main thing to make sure we're all on the same page about is:

* what is the hierarchy/precedence for configuration
* what can be configured, and what is internal and should not be configured

Reply:
> The part that I didn't really understand then was "it's own environment", because in some sense all environments are external...

A yes, that was very bad phrasing, great that you fixed this!

I hope this README can clarify and get us on the same page, as well with everyone else reading it :)

I recognize this from many other software projects and packages btw.. but I think it's understandable that everyone will always struggle a bit with upstream/downstream definitions.